lib/webauthn/authenticator_data/attested_credential_data.rb

```diff
@@ -19,16 +30,14 @@ def public_key_object
         end
       end
 
-      def initialize(data)
-        @data = data
+      def self.deserialize(data)
+        read(data)
+      rescue EOFError
+        raise AttestedCredentialFormatError
```

Comment:
Should this be `WebAuthn::AttestedCredentialDataFormatError`?

Reply:
Yes 🤦‍♂️

Reply:
hehe, no worries